api.js

```diff
@@ -87,6 +87,18 @@ function getArtibleByTopic(topic) {
   });
 }
 
+function sortArticles(sortBy, orderBy) {
+  return axios.get(`https://news-project-1.onrender.com/api/articles?sort_by=${sortBy}&order_by=${orderBy}`).then((response) => {
+    return response.data.articles
+  })
+  .catch((error) => {
+    console.log(error)
```

Comment:
if you don't need this here, I'd take it out. Makes more sense for the catch to be in the useEffect so you can handle it per page.